prime-router/src/main/kotlin/fhirengine/translation/hl7/FhirToHl7Converter.kt

```diff
@@ -87,10 +87,6 @@ class FhirToHl7Converter(
         val message = HL7Utils.getMessageInstance(schemaRef.hl7Class!!)
 
         terser = Terser(message)
-        val dupes = schemaRef.duplicateElements
```

Comment:
Why was this code here in the first place? Do you we have tests that verify what happens when there are duplicate elements?

Reply:
I'll go back and edit the existing tests rather than remove them entirely.